Source/ARTRealtimePresence.m

```diff
@@ -622,4 +660,315 @@ - (void)failPendingPresence:(ARTStatus *)status {
     }
 }
 
+- (void)broadcast:(ARTPresenceMessage *)pm {
+    [_eventEmitter emit:[ARTEvent newWithPresenceAction:pm.action] with:pm];
+}
+
+- (void)sync {
```

Comment:
It's still used in [RTP19](https://sdk.ably.com/builds/ably/specification/main/features/#RTP19):
"..._send a SYNC message with the channel attribute populated with the current channel which will trigger a server initiated SYNC_."

Also is used in a test for `RTP17b` which is skipped though.

Reply:
I don't think it says what you think it says. It's telling to write a local test where we mock server sent `SYNC` message and inject it client side as a part of test.

Reply:
That said, I need to check with @SimonWoolf regarding this

Reply:
Adding @SimonWoolf as a reviewer

Reply:
Okay, got a reply, This is not a public method and should only be used for testing. Can we not make it part of the test suite instead of the codebase. I strongly feel, this is going to be misused if we keep it part of the codebase.

Reply:
Seems we can do this using https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/ProgrammingWithObjectiveC/CustomizingExistingClasses/CustomizingExistingClasses.html.
You can move this code to the test suite, since it;s not a part of the standard spec.

Reply:
If this is a public method, you can add a warning message. I have done the same here
https://github.com/ably/ably-java/pull/989/commits/23513c643b6bd90fd852daeb238f8f948d41c887

Reply:
If this is a public method, add deprecation notice `sync() method is intended only for internal testing purpose as per RTP19`. If not, then move it under test-suite.

Reply:
https://github.com/ably/ably-cocoa/pull/1870/commits/35f43649a5aba361a3fdc7c8cc2dbc35c2247931